pkg/blocks/createfile.go

```diff
@@ -191,7 +87,13 @@ func (s *CreateFileStep) Execute(execCtx TTPExecutionContext) (*ExecutionResult,
 		return nil, err
 	}
 
-	return &ExecutionResult{}, nil
+	return &ActResult{}, nil
+}
+
+func (s *CreateFileStep) GetDefaultCleanupAction() Action {
+	return &RemovePathAction{
```

Comment:
We now support using `cleanup: default` for certain steps - this greatly streamlines TTP creation and reduces how often we need to shell out (avoiding needing to shell out is a core design goal for TTPForge)